lib/openxml/elements/run_properties.rb

```diff
@@ -39,6 +43,10 @@ def initialize(*boolean_properties, **value_properties)
         end
       end
 
+      def font_color=(color)
+        self.push(OpenXml::Elements::RGBColor.new(color))
+      end
```

Comment:
It looks like setting this multiple times will shovel multiple colors into the `rPr` tag. Is that right? — or is it more of a 0-or-1 thing?

Maybe rather than inheriting from Container, this one should be like ... ?

```ruby
attr_accessor :font_color

def to_xml(xml)
  xml[namespace].public_send(tag, xml_attributes) do
    font_color.to_xml(xml) if font_color
  end
end
```